src/config.rs

```diff
@@ -1 +1 @@
-pub const TEST_HOST: &str = "voyager.lemmy.ml";
+pub const TEST_HOST: &str = "localhost:8536";
```

Comment:
does this work with voyager? i am biassed as that is what i am currently using for testing :smile: 

would it be possible to specify protocol in this url (https) as i think that is required for this to work with voyager.

Reply:
I've been preferring localhost since I can look at what gets logged on the server when I come across particularly annoying bugs. As for protocol, I believe I have a method/some logic somewhere that handles that.

Reply:
so i will need to change the code rather than configuration to change the protocol it uses?

Reply:
You can pass in a LEMMY_UI_HTTPS environment variable to have it use HTTPS.